azure-quantum/azure/quantum/qiskit/backends/qci.py

```diff
@@ -52,7 +59,12 @@ def __init__(
 
     @classmethod
     def _default_options(cls) -> Options:
-        return Options(shots=500, targetCapability="AdaptiveExecution")
+        return Options(
+            **{
+                cls._SHOTS_PARAM_NAME: 500 
```

Comment:
we've been setting the default options through `_DEFAULT_SHOTS_COUNT` earlier, we should probably follow the same pattern here.